content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+
+With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors](#processors) in order to extract meaningful information or to extract attributes from semi-structured text to re-use them as [facet](/log/explore/#facets).
+
+Each log that comes through the pipelines is tested against every pipeline filter. If it matches one then all the [processors](#processors) are applied sequentially before moving to the next pipeline.
+
+So for instance a processing pipeline can transform this log:
+{{< highlight json >}}
+{
+"message":"john clicked on a link : www.datadoghq.com for app dd_app",
+"hostname":"host_1",
+"level":"info"
+}
+{{</ highlight >}}
+
+into this log: 
+
+{{< highlight json >}}
+{
+"user":"john",
+"url":"www.datadoghq.com",
+"appname":"dd_app",
+"message":"clicked on a link",
+"hostname":"host_1",
+"severity":"info"
+}
+{{< /highlight >}}
+
+With one single pipeline as follow:
+
+{{< img src="log/processing/pipeline_processors_sketch.png" alt="Pipelines processor sketch" style="width:90%;">}}
+
+Pipelines are also able to take all your logs whatever they source and transform them to handle a unifed and structured format on Datadog.
+
+For instance, in the following example we have at the begining 3 differents raw logs:
+
+{{< img src="log/processing/multi_source_sketch.png" alt="Multi source sketch" >}}
+
+and at the end we have the same information, but with more coherence that allow a better usability.
+
+### Pipeline filters
+
+Attach a filter to your pipeline in order to apply it only to a subset of your logs. 
+
+The filter syntax is the same as the [search bar](/log/explore/#search-bar).
+
+**Be aware that the pipeline filtering is applied before any pipeline processing, hence you can not filter on an attribute that does not exists** 
+
+Check to wich subset of logs your pipeline is applied to in the log list displayed: 
+
+{{< img src="log/processing/pipeline_filters.png" alt="Pipelines processor sketch" style="width:90%;">}}
+
+### Integrations Pipelines
+
+Integration pipelines are automatically installed when you activate an [integration](https://app.datadoghq.com/account/settings)
+
+Those pipeline are read-only but can be cloned for further customisation:
+
+{{< img src="log/processing/cloning_pipeline.png" alt="Cloning pipeline" >}}
+
+## Processors
+
+**A processor executes a well identified action on a log most often related to parsing & enrichment.**
+
+The different kind of processors are explained below.
+
+### Grok Parser
+
+Create custom grok rules to parse the full message or a specific attribute of your raw event:
+
+{{< img src="log/processing/parser.png" alt="Parser" >}}
+
+Find more about this in our dedicated [parsing section](/log/parsing)
+
+### Log Date Remapper 
+
+By default (as explained [here](/log/#the-date-attribute)) Datadog has a couple of reserved attributes that are, by default, considered as the date reference for the main timeline.
```

Comment:
> Datadog has a couple of reserved attributes

And what are those attributes? (I guess it's explained in /log/#the-date-attribute? If that page isn't too long, should we just include it here?)

Suggested rewrite (tell me if this is incorrect—this is my understanding from reading this): 

"As Datadog receives logs, it timestamps them using the value(s) from any of these default attributes:

<list the attributes>

If your logs put their dates in an attribute not in this list, use the log date remapper processor to define their date attribute as the official log timestamp.

If your logs don't contain any of the default attributes and you haven't defined your own date attribute, Datadog timestamps the logs with the date it received them."

Reply:
Perfect comprehension, I'm going to inline the list of attributes. good idea 👍